recipes/virstrain/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "virstrain" %}
+{% set version = "1.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 423bd42087c9b49d24ec1ccd69bc474e7fa17c6ab322ab39a4d1b52327a5921b
+
+build:
+  number: 0
+  entry_points:
+    - virstrain = VirStrain.VirStrain:main
+    - virstrain_build = VirStrain.VirStrain_build:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - biopython ==1.74
+    - matplotlib-base ==3.1.2
+    - networkx ==2.4
+    - numpy ==1.17.3
+    - pandas ==1.0.1
+    - pip
+    - plotly ==3.10.0
+    - python ==3.7.3
```

```suggestion
    - pip
    - python
```
you should only need python and pip in the host section

---

Fixed, thanks!